js/libs/templates.js

```
@@ -27,9 +27,63 @@
 }
 ];
 
-Fliplet.FormBuilder.templates = function() {
+// Cache for organization templates
+var templateCache = {
+  key: 'fliplet-form-templates-cache',
+  ttl: 5 * 60 * 1000, // 5 minutes
+
+  get: function(organizationId) {
+    try {
+      var cached = sessionStorage.getItem(this.key + '-' + organizationId);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)